packages/dev/core/src/Engines/thinEngine.ts

```diff
@@ -1391,6 +1392,11 @@ export class ThinEngine {
         for (let slot = 0; slot < this._maxSimultaneousTextures; slot++) {
             this._nextFreeTextureSlots.push(slot);
         }
+
+        if (this._glRenderer === "Mali-G72") {

```

Comment:
is there a way to move it with the rest of the exceptions ?

Reply:
The other exceptions (if you mean the parsing of `ThinEngine.ExceptionList`) are handled in the constructor, before the caps are built and before the WebGL context is retrieved, so I'm not sure it can be moved. The current exception list is not based on data we would retrieve from the WebGL context, they are based only on the current user agent.